internal/provider/resource_consul_cluster_test.go

```diff
@@ -186,7 +189,7 @@ func TestAccConsulCluster(t *testing.T) {
 				Config: testConfig(setTestAccConsulClusterConfig(updatedConsulCluster)),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckConsulClusterExists(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "size", "MEDIUM"),
+					resource.TestCheckResourceAttr(resourceName, "size", "small"),
```

Comment:
This causes an error in the test: `error="Check failed: Check 2/2 error: hcp_consul_cluster.test: Attribute 'size' expected \"small\", got \"SMALL\""`
```suggestion
					resource.TestCheckResourceAttr(resourceName, "size", "SMALL"),
```